asteroidsyncserviced/telescope.pri

```diff
@@ -0,0 +1,29 @@
+
+SOURCES += platforms/ubuntutouch/main.cpp \
+    watchesmanager.cpp \
```

Comment:
Could you keep the shared source files between telescope and starfish in the SOURCES and HEADERS variables of the .pro file? And only keep SOURCES += for the platform specific files?

I think you can also factorize some other variables such as INSTALLS etc...

Reply:
done